to_code/ex_11.py

```diff
@@ -6,12 +6,40 @@
 def ex11_1():
     nums = []
     for i in range(1, 11):
-        num = int(input("Please enter a number #%d: " % i))
+        # 1. Using %d is a bit outdated. Today it is considered more modern to use f{} strings:
+        # Please see lesson here:
+        # https://www.tocode.co.il/bundles/advanced-python3/lessons/python3-cool-features
+        num = int(input(f"Please enter a number #{i}: "))
         nums.append(num)
-    print("The biggest number is: %d" % max(nums))
+    print(f"The biggest number is: {max(nums)}")
```

Comment:
Although using `%d` is a little bit old-fashioned, I like the `format` method more than the f-string and I use it a lot. I very rarely use f-strings like @ynonp suggested. With `format` there are 3 options: You can either use `{}`, or `{0}` with a number, or `{name}` with a variable name. It depends how complicated is your code and if you translate it, for example `{0}` may come after `{1}` in the translated string. But the simplest thing is to use `{}`:

```python
print("The biggest number is: {}".format(max(nums)))
```